bot/adapters/rest/rest_responder.py

```diff
@@ -0,0 +1,83 @@
+from pathlib import Path
+
+from fastapi.responses import (
+    FileResponse,
+    JSONResponse,
+)
+
+from bot.interfaces.responder import AbstractResponder
+
+
+class RestResponder(AbstractResponder):
+    def __init__(self):
+        self.responses = []
+        self._response_given = False
+
+    async def send_text(self, text: str):
+        if self._response_given:
+            return
+        self.responses.append({
+            "type": "text",
+            "content": text,
+        })
+        self._response_given = True
+
+    async def send_markdown(self, text: str):
+        if self._response_given:
+            return
+        self.responses.append({
+            "type": "markdown",
+            "content": text,
+        })
+        self._response_given = True
+
+    async def send_photo(self, image_bytes: bytes, image_path: Path, caption: str):
+        if self._response_given:
+            return
+        self.responses.append({
+            "type": "photo",
+            "caption": caption,
+            "filename": str(image_path),
+        })
+        self._response_given = True
+
+    async def send_video(self, file_path: Path):
+        if self._response_given:
+            return
+        self.responses.append({
+            "type": "video",
+            "filename": str(file_path),
+        })
+        self._response_given = True
+
+    async def send_document(self, file_path: Path, caption: str):
+        if self._response_given:
+            return
```

Comment:
Serio mamy przypadek gdzie mogłoby się wydarzyć ponowne wywołanie respondera? trochę krzywa logika jeżeli to jest potrzebne, a jeśli nie jest to do wywalenia